docs/tutorials/add-prisma-database.md

```diff
@@ -0,0 +1,344 @@
+
+# External Prisma database support for a Seven add-on
+
+This tutorial will show you how to add a small "Like" counter button to any Seven content object.
+It will persist its value in an external database using Prisma.
+It also demonstrates how to create API routes and slots in Seven and how to share data from the server to the client through the root loader.
+In this tutorial you will:
+
+- install Prisma inside an existing Seven add-on
+- model a `UrlLike` table backed by SQLite for local development
+- expose API endpoints that read and update the like counter
+- surface the latest count in a slot component rendered on every page
+
+Prisma is an open-source toolkit that simplifies database access.
+You can always fall back to the official docs for deeper dives: https://www.prisma.io/docs/.
+
+```{note}
+The main purpose of this tutorial is to demonstrate how to integrate Prisma with Seven.
+You can also use the same approach but saving to a Plone REST API endpoint.
+It is not meant to be a complete, production-ready implementation of a like button.
+You can find the complete code of this tutorial in this [GitHub repository](https://github.com/collective/seven-training-addon).
+```
+
+## Prerequisites
+
+- A Seven add-on scaffolded with {term}`cookieplone`
+- {term}`pnpm` installed and configured for the workspace
+- Basic familiarity with TypeScript/React Router code in a Seven project
+
+If you still need an add-on skeleton, follow {doc}`../get-started/create-package` before you continue.
+
+## Install Prisma in your add-on
+
+Run this command from the repo root to install Prisma and its client in your add-on package:
+
+```shell
+pnpm --filter <addon-name> add @prisma/client prisma @types/node
+```
+
+```{warning}
+{term}`pnpm` will require you to approve the builds of this packages, by running `pnpm approve-builds` command.
+Follow the instructions in the terminal to approve the builds.
+
+```{image} ../_static/approve-builds.jpg
+:alt: pnpm approve-builds
+:width: 700px
+:align: center
+```
+
+In your add-on's `package/<addon-name>/package.json`, add these convenience scripts to work with the Prisma client:
+
+```{code-block} json
+:caption: packages/\<addon-name>/package.json
```

Comment:
Interesting. I didn't know you need to escape `<` in captions. It must be a reStructuredText artifact.

```suggestion
:caption: packages/\<add-on-name>/package.json
```